src/patternfly/components/List/list.scss

```diff
@@ -1,54 +1,39 @@
 @import "../../patternfly-imports";
 
 .pf-c-list {
-  // Inline modifier variables
-  --pf-c-list--m-inline--MarginRight: var(--pf-global--spacer--md);
-
-  // Grid modifier variables
-  --pf-c-list--m-grid--GridGap: var(--pf-global--spacer--md);
-
-  // Content variables (no modifier)
-  --pf-c-content--li--MarginTop: var(--pf-global--spacer--sm);
-  --pf-c-content--ul--PaddingLeft: var(--pf-global--spacer--lg);
-  --pf-c-content--ul--MarginTop: var(--pf-global--spacer--md);
-  --pf-c-content--MarginBottom: var(--pf-global--spacer--md);
-  --pf-c-content--ul--MarginLeft: var(--pf-global--spacer--lg);
-  --pf-c-content--ul--ListStyle: var(--pf-global--ListStyle);
-  --pf-c-content--ul--nested--MarginTop: var(--pf-global--spacer--sm);
-  --pf-c-content--ul--nested--MarginLeft: var(--pf-global--spacer--sm);
-
-  &.pf-m-inline {
-    display: flex;
-    flex-wrap: wrap;
+  --pf-c-list--PaddingLeft: var(--pf-global--spacer--lg);
+  --pf-c-list--ul--nested--MarginTop: var(--pf-global--spacer--sm);
+  --pf-c-list--ul--nested--MarginLeft: var(--pf-global--spacer--sm);
+  --pf-c-list--ul--ListStyle: var(--pf-global--ListStyle);
+  --pf-c-list--li--MarginTop: var(--pf-global--spacer--sm);
+  --pf-c-list--m-inline--li--MarginRight: var(--pf-global--spacer--lg);
+
+  padding-left: var(--pf-c-list--PaddingLeft);
+
+  ol,
+  ul {
+    margin-top: var(--pf-c-list--ul--nested--MarginTop);
```

Comment:
Just a thought, does this variable need ul in its name since list covers ol and ul? So you could write it as --pf-c-list--nested--MarginTop?

Reply:
good call, I like it :)